src/dataflow/workflows/nightscout/dagster_job.py

```
@@ -1,0 +1,273 @@
+    },
+)
+@job(
+    name="nightscout_job",
+    description="Job for ingesting and transforming Nightscout data",
+    resource_defs={
+        "nightscout_config": NightscoutResource.configure_at_launch(),
+    },
+)
+def nightscout_job():
+    """Dagster job for ingesting and processing Nightscout data."""
+    transformation_result = nightscout_transform_op(nightscout_ingestion_op())
```

Ruff (0.8.2)

Local variable `transformation_result` is assigned to but never used

Remove assignment to unused variable `transformation_result`

(F841)